src/applications/vaos/new-appointment/components/ClinicChoicePage/index.jsx

```diff
@@ -137,7 +137,7 @@ export default function ClinicChoicePage() {
       >
         {usingUnsupportedRequestFlow && (
           <div className="vads-u-margin-top--2">
-            <EligibilityCheckMessage
+            <RequestEligibilityMessage
```

Comment:
I renamed this because it only shows request related messages, since if you're on this page you already passed the DS checks.